packages/dashboard/src/components/Card/animations/BarGraph.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+
+import { mergeStyles } from 'office-ui-fabric-react/lib/Styling';
+
+import { getStyles } from './BarGraph.style';
+
+export interface IBarGraphProps {}
+
+export class BarGraph extends React.Component<IBarGraphProps> {
+  constructor(props: IBarGraphProps) {
+    super(props);
+  }
+
+  public render(): JSX.Element {
+    const barGraphLoadingClassName = mergeStyles(getStyles().barGraphLoading);
```

Comment:
Please use the classNames function approach used in other components rather than repeatedly calling mergeStyles and getStyles.